### PR TITLE
Add simple-duck implementation link

### PR DESCRIPTION
**Pull request by xander27**
I wasnt happy with existing implementations of duck idea, so created my own. Code looks pretty explicit and simple. Already used it couple projects

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Although it's completely feasable to implement it without any extra library, the
  * [modular-redux-thunk](https://github.com/benbeadle/modular-redux-thunk) - A ducks-inspired package to help organize actions, reducers, and selectors together - with built-in redux-thunk support for async actions.
  * [molecular-js](https://www.npmjs.com/package/molecular-js) - Set of utilities to ease the development of modular state management patterns with Redux (also known as ducks).
  * [ducks-reducer](https://github.com/drpicox/ducks-reducer) - Function to combine _ducks object_ reducers into one reducer (equivalent to [combineReducers](https://redux.js.org/docs/api/combineReducers.html)), and function [ducks-middleware](https://github.com/drpicox/ducks-middleware) to combine _ducks object_ middleware into one single middleware compatible with [applyMiddleware](https://redux.js.org/docs/api/applyMiddleware.html).
+ * [simple-duck](https://github.com/xander27/simple-duck) - Class based implementation of modules system, inspired by duck-modolular-redux. All OOP benifits like inheritance and composition. Support combining of duck-module classes and regular reducer functions using `combineModules` function.
```
Comment (jdreesen) [inline README.md]: There are some typos: 
- `duck-modolular-redux` -> `ducks-modular-redux`
- `benifits` -> `benefits`

Comment (xander27) [reply README.md]: Sorry. Not native english. Spellchecked now

Comment (xander27) [reply README.md]: Any other changes needed?

**Comment by xander27** [comment]
Do I need to fix something else?

**Pull request merged**